packages/palette/src/elements/MultiSelect/MultiSelect.tsx

```diff
@@ -39,10 +42,13 @@ export const MultiSelect: React.FC<MultiSelectProps> = ({
   required,
   title,
   onSelect,
+  prefixOffset,
+  suffixOffset,
```

Comment:
What purpose would these props serve? Doesn't seem needed.

Reply:
This PR is still in draft form and not ready for review. I'm still technically working out how to approach the styling with a lot of difficulty on my end.